static/src/stylesheets/module/commercial/gimbap/_gimbap-richmedia.scss

```diff
@@ -0,0 +1,72 @@
+%gimbap-zindex {
+    position: relative;
+    z-index: 1;
+}
+
+.gimbap-richmedia__title {
+    @extend %gimbap-zindex;
+    @include fs-headline(3);
```

Comment:
I've been using `get-font-size` recently, especially since most of the time I overwrite `font-weight`, `line-height`, etc.